src/test/java/se/sundsvall/billingpreprocessor/integration/db/BillingRecordRepositoryTest.java

```diff
@@ -43,6 +22,24 @@
 import se.sundsvall.billingpreprocessor.integration.db.model.InvoiceRowEntity;
 import se.sundsvall.billingpreprocessor.integration.db.model.IssuerEntity;
 
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.time.OffsetDateTime.now;
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static java.util.stream.Collectors.toCollection;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.assertj.core.api.Assertions.within;
+import static se.sundsvall.billingpreprocessor.api.model.enums.Status.APPROVED;
+import static se.sundsvall.billingpreprocessor.api.model.enums.Status.NEW;
+import static se.sundsvall.billingpreprocessor.api.model.enums.Status.REJECTED;
+import static se.sundsvall.billingpreprocessor.api.model.enums.Type.EXTERNAL;
+import static se.sundsvall.billingpreprocessor.integration.db.model.DescriptionType.DETAILED;
+import static se.sundsvall.billingpreprocessor.integration.db.model.DescriptionType.STANDARD;
+
 /**
  * billingRecord repository tests
  *
```

Comment:
`@DataJpaTest` instead?

Reply:
Then FilterSpecificationConverter is not availiable.